.travis.yml

```diff
@@ -1,7 +1,7 @@
 language:
   - node_js
 node_js:
-  - 9.5.0
+  - 13.3.0
```

Comment:
In my opinion this shouldn't be updated. We are supporting from version 9.5.0. Or change the documentation.